dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-github-dynamic/src/dynamic/index.ts

```diff
@@ -4,20 +4,16 @@ import { GithubEntityProvider } from '@backstage/plugin-catalog-backend-module-g
 export const dynamicPluginInstaller: BackendDynamicPluginInstaller = {
   kind: 'legacy',
   async catalog(builder, env) {
-    if (env.config.getOptionalBoolean('enabled.github')) {
-      builder.addEntityProvider(
-        GithubEntityProvider.fromConfig(env.config, {
-          logger: env.logger,
-          schedule: env.scheduler.createScheduledTaskRunner({
-            frequency: { minutes: 30 },
-            timeout: { minutes: 3 },
-            initialDelay: { minutes: 1 },
-          }),
-          scheduler: env.scheduler,
+    builder.addEntityProvider(
+      GithubEntityProvider.fromConfig(env.config, {
+        logger: env.logger,
+        schedule: env.scheduler.createScheduledTaskRunner({
+          frequency: { minutes: 30 },
+          timeout: { minutes: 3 },
+          initialDelay: { minutes: 1 },
         }),
-      );
-    } else {
-      env.logger.info('Github plugin is disabled');
-    }
+        scheduler: env.scheduler,
```

Comment:
I think the upstream github plugin has the same behavior where the [hard coded `schedule` field takes precedence over the `scheduler`](https://github.com/backstage/backstage/blob/438f7f87ac61095ae700e085a375c41cf16f5c58/plugins/catalog-backend-module-github/src/providers/GithubEntityProvider.ts#L112-L114). This would require a change in the upstream.